app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+
+$f-pnav__main--zindex: 1030;
+$f-pnav__side--width: 320px;
+$f-pnav__side--zindex: 1032;
+
+@mixin deals_link_base() {
+  color: $color-forever_orange;
+
+  &:hover {
+    color: $color-forever_blue;
+  }
+
+  &::after {
+    position: relative;
+    top: -5px;
+    right: -2px;
+    font-family: FontAwesome;
+    font-size: 10px;
+    content: '\f005';
+  }
+}
+
+.f-pnav {
```

Comment:
I'm not sure we need to preface everything w/ .f- ? is that to distinguish from bootstrap or?

Reply:
We've always prefaced everything in mars to try and guarantee uniqueness and prevent conflicts.

Reply:
Probably a minor concern, but the possibility exists and could lead to confusion.